site/operational-guide/npc.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Weave NPC
+menu_order: 70
+---
+
+* ["Connection dropped by Weave NPC"](#connection-dropped)
+
+##<a name="connection-dropped"></a>"Connection dropped by Weave NPC"
+
+When using Weave together with [Kubernetes](http://kubernetes.io) and [Weave NPC](https://github.com/weaveworks/weave-npc), Weave's Kubernetes Network Policy Controller, `weave launch --expect-npc` sets up the two below default `iptables` rules, in case none of the NPC rules apply:
```

Comment:
I'm not sure we want to include the details like `--expect-npc` here.  Introduce the subject, talk generally about how it works, then maybe add a "how it works" section at the end.